app/views/home/_oracle.html.erb

```diff
@@ -0,0 +1,54 @@
+<%= tag.h4 "We couldn't find a recipe that matches your inventory. :(", class: "subtitle has-text-centered" unless @doable_recipes.length > 0 %>
```

Comment:
You can also use `empty?` e.g: `if @doable_recipes.empty?`. Same effect, tho.

Reply:
Even more clear 👆🏻